.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the development branch
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    
+  push:
```

Comment:
Is this needed because this will go away. pull_request is not sufficient?

Reply:
Maybe needed until we go to master?

Reply:
Yes, I apply it on this branch until everything is merged into master
Then we'll just have to apply tests to master